### DISCUSSION TITLE
[Informationssystem] Hinzufügen von News als Admin

### DISCUSSION THREAD

**Discussion by laralngn** [SE-TINF22B2/G2-Schoolify#38] [User Stories]
### User Story:
Als Admin möchte ich aktuelle Nachrichten mit Text und ggf. Bild auf der Startseite posten können, um Andere auf dem neusten Stand zu halten.

### Beschreibung:
Auf der Startseite sollen Admins einen Button haben, um in den Editor Modus zu kommen. Über diesen sollen sie einen Text und ggf. ein Bild hinzufügen können, welche als neuer Post gespeichert werden kann. Dieser Post soll dann für alle auf der Startseite im News-Bereich sichtbar sein.

### Akzeptanz-Kriterien:
- [ ] Button für Editor Modus
- [ ] Admins sollen im Editor Modus ein Eingabefeld und Bild-Upload haben
- [ ] Post soll bei aktuellen News auf der Startseite gepostet werden

**Comment by laralngn** [comment]
Brauchen wir dieses Feature noch? @SE-TINF22B2/tinf22b2-g2

**Reply by tilwbr** [reply]
ist ja quasi die news seite oder? würde sagen ist schon wichtig @SE-TINF22B2/tinf22b2-g2

**Reply by Schmarvinius** [reply]
Ja genau das ist im Prinzip die Schüler Zeitung 
Finde das ist relativ wichtig

Frage die sich daraus ableitet:
Sollen Schüler, mit zugewiesenen Rollen, die News Artikel selber posten oder soll das über den Admin (Sekretariat) an den Server gesendet werden?

Finde über Admin besser :)

**Reply by tilwbr** [reply]
würde auch sagen übre Admin ist besser, sonst können die ja jede scheiße hochladen
